util/reggen/clocking.py

```diff
@@ -49,6 +54,16 @@ def _asdict(self) -> Dict[str, object]:
         ret['primary'] = self.primary
         return ret
 
+    def get_base_name(self) -> str:
+        if not self.clock:
+            raise ValueError('Cannot extract clock base name from None clock')
```

Comment:
Maybe something like "Cannot extract clock base name: this clocking item has no clock" ? The current text looks a bit like we're calling a method a "None clock", which isn't quite what's going on.